### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by AyeMolo** [GiorgioBrux/nitro-sniper-enhanced#156] [Q&A]
How do you bypass this "We couldn't deploy your app because the source code violates the Salesforce Acceptable Use and External-Facing Services Policy."

**Comment by GiorgioBrux** [comment]
Apologies for not answering earlier, github doesn't send me notifications for discussions.
Did you try the steps listed [here](https://github.com/GiorgioBrux/nitro-sniper-enhanced#heroku-recommended)?
